### ISSUE TITLE
[BUG] 1.8.4 Building failed

### ISSUE THREAD

**Issue by MotiasF33** [TheRatCode/RatPoison#730]
### Describe the bug:

**Building failed:**
* Missing installer, try auto-download causes - TLS/SSL Error - "Could not create SSL/TLS secure channel"
**several ways to solve** 
* Can't download 1.9.13-SNAPSHOT - Version has been changed
**added pull request**

### To Reproduce:
1. [failure] Start installer.bat
**Missing installer, try auto-downloading it - error**
```
Exception calling "DownloadFile" with "2" argument(s): "The request was aborted:
Could not create SSL/TLS secure channel."
At line:1 char:1
+ (New-Object Net.WebClient).DownloadFile('http://github.com/SPRAVEDLIV ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : WebException
```
2. [success] Downloading [installer.exe](https://github.com/TheRatCode/RatPoison/blob/master/installer.exe) from _master_ branch 
3. [success] Start installer.bat
3.1. [failure] Good! downloading, building, but failed.
**Can't download 1.9.13-SNAPSHOT**
```
   > Could not determine the dependencies of task ':shadowJar'.
   > Could not find com.badlogicgames.gdx:gdx-box2d:1.9.13-SNAPSHOT.
   > Could not find com.badlogicgames.gdx:gdx-backend-lwjgl3:1.9.13-SNAPSHOT.
   > Could not find com.badlogicgames.gdx:gdx-platform:1.9.13-SNAPSHOT.
   > Could not find com.badlogicgames.gdx:gdx-box2d-platform:1.9.13-SNAPSHOT.
   > Could not find com.badlogicgames.gdx:gdx-freetype-platform:1.9.13-SNAPSHOT.
   > Could not find com.badlogicgames.gdx:gdx-freetype:1.9.13-SNAPSHOT.
   > Could not find com.badlogicgames.gdx:gdx:1.9.13-SNAPSHOT.
```
4. [failure] Trying downloading last release installer.exe using [link]( https://github.com/SPRAVEDLIVO/RatInstaller/releases/download/v2.11.6) from installer.bat 
**Page not found - Error 404**
5. [success] Downloading installer.exe manualy [from](https://github.com/SPRAVEDLIVO/RatInstaller/releases/download/v2.11.6/installer.exe) 
6. [success] Start installer.bat
6.1. [failure] Error same as 3.1
**Can't download 1.9.13-SNAPSHOT**
7. [success] Going [to](https://oss.sonatype.org/content/repositories/snapshots/com/badlogicgames/gdx/gdx/)
**We see that the version of the snapshot has been changed to 1.9.15**
8. [success] Change version to _1.9.15_ in [build.gradle](https://github.com/TheRatCode/RatPoison/blob/beta/build.gradle)
9. [success] Start installer.bat
10. [success] Build Successful.

**Comment by sotakoira** [comment]
Edit [this line](https://github.com/TheRatCode/RatPoison/blob/beta/build.gradle#L7) to `ext.gdxVersion = '1.9.13'`

**Comment by MotiasF33** [comment]
Possibly sollution add [Net.ServicePointManager]::SecurityProtocol = "tls12"
but it doesn't work for me( 
```
PS C:\Users\User> [Net.ServicePointManager]::SecurityProtocol = "Ssl3, Tls, Tls11, Tls12, Tls13"
PS C:\Users\User> [Net.ServicePointManager]::SecurityProtocol
Ssl3, Tls, Tls11, Tls12, Tls13
PS C:\Users\User> powershell -Command "(New-Object Net.WebClient).DownloadFile('http://github.com/SPRAVEDLIVO/RatInstaller/releases/
latest/download/installer.exe', 'installer.exe')"
Exception calling "DownloadFile" with "2" argument(s): "The request was aborted: Could not create SSL/TLS secure channel."
At line:1 char:1
+ (New-Object Net.WebClient).DownloadFile('http://github.com/SPRAVEDLIV ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : WebException
```

**Comment by RamyS2K** [comment]
I have the same problem, it always download openjdk-14.0.2_windows-x64_bin.zip and then says "some exception occurred"

I tried asking in discord server, but ain't work

**Comment by MotiasF33** [comment]
I

> 
> 
> I have the same problem, it always download openjdk-14.0.2_windows-x64_bin.zip and then says "some exception occurred"
> 
> I tried asking in discord server, but ain't work

I solved it, follow for my steps

**Comment by RamyS2K** [comment]
> I
> 
> > I have the same problem, it always download openjdk-14.0.2_windows-x64_bin.zip and then says "some exception occurred"
> > I tried asking in discord server, but ain't work
> 
> I solved it, follow for my steps

I redownload the 1.8.4 beta zip file and open installer.bat and it says "Installer script wasn't found. Downloading..." and then start downloading openjdk-14.0.2_windows-x64_bin.zip again. where do i save it?

![image](https://user-images.githubusercontent.com/64722734/110064073-5f7d7500-7da7-11eb-8bff-5c379939ba8c.png)

**Comment by MotiasF33** [comment]
> 
> 
> > I
> > > I have the same problem, it always download openjdk-14.0.2_windows-x64_bin.zip and then says "some exception occurred"
> > > I tried asking in discord server, but ain't work
> > 
> > 
> > I solved it, follow for my steps
> 
> I redownload the 1.8.4 beta zip file and open installer.bat and it says "Installer script wasn't found. Downloading..." and then start downloading openjdk-14.0.2_windows-x64_bin.zip again. where do i save it?
> 
> ![image](https://user-images.githubusercontent.com/64722734/110064073-5f7d7500-7da7-11eb-8bff-5c379939ba8c.png)

im updated my issue, follow for my _successful_ steps

**Comment by RamyS2K** [comment]
> 4. [success] Going [to](https://oss.sonatype.org/content/repositories/snapshots/com/badlogicgames/gdx/gdx/)
>    **We see that the version of the snapshot has been changed to 1.9.15**
> 5. [success] Change version to _1.9.15_ in [build.gradle](https://github.com/TheRatCode/RatPoison/blob/beta/build.gradle)
> 6. [success] Start installer.bat
> 7. [success] Build Successful.
I have followed these step, it still give me Build Failed error 
![image](https://user-images.githubusercontent.com/64722734/110065982-6b6b3600-7dab-11eb-8f04-1cfe12edb3ec.png)

**Comment by RamyS2K** [comment]
> > 1. [success] Going [to](https://oss.sonatype.org/content/repositories/snapshots/com/badlogicgames/gdx/gdx/)
> >    **We see that the version of the snapshot has been changed to 1.9.15**
> > 2. [success] Change version to _1.9.15_ in [build.gradle](https://github.com/TheRatCode/RatPoison/blob/beta/build.gradle)
> > 3. [success] Start installer.bat
> > 4. [success] Build Successful.
> >    I have followed these step, it still give me Build Failed error
> >    ![image](https://user-images.githubusercontent.com/64722734/110065982-6b6b3600-7dab-11eb-8f04-1cfe12edb3ec.png)

UPDATE: ok i manage to fix it by change version to 1.9.13 in build.gradle

still, thanks to MotiasF33 for giving me detailed steps
